examples/with-tailwind/apps/docs/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "docs",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev --port 3001",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "latest",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "ui": "*"
+  },
+  "devDependencies": {
+    "@types/node": "^17.0.12",
+    "@types/react": "^17.0.37",
+    "@types/react-dom": "^17.0.11",
+    "autoprefixer": "^10.4.0",
+    "eslint-config-custom": "*",
+    "postcss": "^8.4.4",
+    "tailwind-config": "*",
+    "tailwindcss": "^3.0.0",
```

Comment:
`tailwind-config` is already using `tailwindcss` as a devDependency, do we need to have `tailwindcss` explicitly here?

https://github.com/vercel/turbo/blob/main/examples/with-tailwind/packages/tailwind-config/package.json#L7